modules/deprecated/graphql/graphql.info.yml

```diff
@@ -0,0 +1,7 @@
+name: GraphQL
```

Comment:
@IvanZugec Move the stub module into https://github.com/govCMS/lagoon/tree/9.x/modules